documentation/basic_concepts.md

```diff
@@ -113,6 +113,13 @@ Built-in middlewares:
 2. `Crawly.Middlewares.RobotsTxt` - this middleware ensures that Crawly respects the robots.txt defined by the target website.
 3. `Crawly.Middlewares.UniqueRequest` - this middleware ensures that crawly would not schedule the same URL(request) multiple times.
 4. `Crawly.Middlewares.UserAgent` - this middleware is used to set a User Agent HTTP header. Allows to rotate UserAgents, if the last one is defined as a list.
+5. `Crawly.Middlewares.RequestOptions` - allows to set additional request options, for example timeout, of proxy string.
+   
+   
```

Comment:
Should we add in a blurb that says that the options handling behaviour is handled by indivudls fetcher modules? For e.g. For a HTTPoison module, the options should match HTTPoison's request options

Reply:
yes, make sense